src/dash-table/derived/cell/eventHandlerProps.ts

```diff
@@ -30,11 +30,11 @@ const getter = (propsFn: () => ICellFactoryProps): CacheFn => {
         ] = args;
 
         return {
-            onChange: derivedHandlers(Handler.Change, rowIndex, columnIndex),
-            onClick: derivedHandlers(Handler.Click, rowIndex, columnIndex),
-            onDoubleClick: derivedHandlers(Handler.DoubleClick, rowIndex, columnIndex),
-            onMouseUp: derivedHandlers(Handler.MouseUp, rowIndex, columnIndex),
-            onPaste: derivedHandlers(Handler.Paste, rowIndex, columnIndex)
+            onChange: derivedHandlers(Handler.Change, columnIndex, rowIndex),
+            onClick: derivedHandlers(Handler.Click, columnIndex, rowIndex),
+            onDoubleClick: derivedHandlers(Handler.DoubleClick, columnIndex, rowIndex),
+            onMouseUp: derivedHandlers(Handler.MouseUp, columnIndex, rowIndex),
+            onPaste: derivedHandlers(Handler.Paste, columnIndex, rowIndex)
```

Comment:
Inverted params order to better match what was done elsewhere.
This and the above eventHandler will probably be merged in the future -- it just impacts a little too many things for now.